lib/elixir/pages/Deprecations.md

```diff
@@ -17,6 +17,7 @@ Deprecated feature                               | Deprecated in | Replaced by (
 :----------------------------------------------- | :------------ | :----------------------------
 `%{key => value}` in typespecs                   | [v1.6]        | `%{required(key) => value}` or `%{optional(key) => value}` (v1.4 and OTP 19)
 `Enum.partition/2`                               | [v1.6]        | `Enum.split_with/2` (v1.4)
+`<%=` in middle and end expressions              | [v1.5]        | `<%=` in the start expression (v1.0)
```

Comment:
We should mention it belongs to EEx, and it should be placed at the end of the v1.5 deprecations list.